feat(genapi): mention unsupported fields for gemma4 (#6576)
scaleway/docs-content: pages/generative-apis/reference-content/supported-models.mdx

| Supports structured output | Yes  |
| Supports function calling |  Yes |
| Supports parallel tool calling | Yes |
| Supported reasoning efforts\* | `none`, `low`, `medium`, `high` |
| Supported image formats | PNG, JPEG, WEBP, and non-animated GIFs |
| Maximum image resolution (pixels) | 896x896 |
| Token dimension (pixels)| 64x64 |
| Supported languages | English, Chinese, Japanese, Korean, and 136 additional languages  |
| Compatible Instances (max context in tokens\*\*) - Dedicated Deployment | H100 (66k), H100-2 (262k), H100-SXM-2 (262k), H100-SXM-4  |
| Hugging Face model card | [google/gemma-4-31B-it](https://huggingface.co/google/gemma-4-31B-it) |

> \*Some providers recommend to enable reasoning using additional arguments such as `"chat_template_kwargs": {"enable_thinking": true}`. These field are not supported on Generative APIs and we recommand using `reasoning_effort` property which is equivalent.

Reviewer: ```suggestion
\*Some providers recommend enabling reasoning by using additional arguments such as `"chat_template_kwargs": {"enable_thinking": true}`. These fields are not supported on Generative APIs, and we recommend using the `reasoning_effort` property, which is equivalent.
```

| Supports structured output | Yes  |
| Supports function calling |  Yes |
| Supports parallel tool-calling | Yes |
| Supported reasoning efforts\* | `none`, `low`, `medium`, `high` |
| Supported image formats | PNG, JPEG, WEBP, and non-animated GIFs |
| Maximum image resolution (pixels) | 896x896 |
| Token dimension (pixels) | 64x64 |
| Supported languages | English, Chinese, Japanese, Korean, and 136 additional languages |
| Compatible Instances (max context in tokens\*\*) - Dedicated Deployment | H100 (262k), H100-2 (262k), H100-SXM-2 (262k), H100-SXM-4 |
| Hugging Face model card | [gemma-4-26B-A4B-it](https://huggingface.co/google/gemma-4-26B-A4B-it) |

> \*Some providers recommend to enable reasoning using additional arguments such as `"chat_template_kwargs": {"enable_thinking": true}`. These field are not supported on Generative APIs and we recommand using `reasoning_effort` property which is equivalent.

Reviewer: ```suggestion
\*Some providers recommend enabling reasoning by using additional arguments such as `"chat_template_kwargs": {"enable_thinking": true}`. These fields are not supported on Generative APIs, and we recommend using the `reasoning_effort` property, which is equivalent.
```
